public/index.html

```diff
@@ -51,20 +52,41 @@
 
 	        </div>
 	    </div>
+        
+        <div id="todo-form" class="row">
+            <div class="col-sm-8 col-sm-offset-2 text-center">
+                <form>
+                    <div class="form-group">
+                        <input type="text" class="form-control input-lg text-center" placeholder="I want more todos" ng-model="newTodo.text">
+                    </div>
 
-		<div id="todo-form" class="row">
-			<div class="col-sm-8 col-sm-offset-2 text-center">
-				<form>
-					<div class="form-group">
-						<input type="text" class="form-control input-lg text-center" placeholder="I want more todos" ng-model="newTodo.text">
-					</div>
+                    <button type="submit" class="btn btn-primary btn-lg" ng-click="createTodo()">Add</button>
+                </form>
+            </div>
+        </div>
 
-					<button type="submit" class="btn btn-primary btn-lg" ng-click="createTodo()">Add</button>
-				</form>
-			</div>
-		</div>
+	    <div class="jumbotron text-center">
+            <h5>Already done :) <span class="label label-info">{{ dones.length }}</span></h5>
+        </div>
+
+	    <div id="finished-todo-list" class="row">
+            <div class="col-sm-4 col-sm-offset-4">
+
+                <div class="checkbox" ng-repeat="done in dones">
+                    <del>
+                        <label>
+                            <input type="checkbox" ng-model="done.done" ng-click="switchTodo(done)"> {{ done.text }}
```

Comment:
`done.done` - it is very complicated to understand what is `done` and what is `done.done` :)